g2core/canonical_machine.cpp

```diff
@@ -1370,6 +1370,13 @@ stat_t cm_change_tool(const uint8_t tool_change)
     return (STAT_OK);
 }
 
+bool cm_is_laser_tool(void)
+{
+    // Tool 32 is always a laser.  Later we can use a different qualified.
```

Comment:
Nit picking here, but is this sentence supposed to have more words? :smile:

Reply:
Typo.  **qualified** should have been **qualifier**.  I will fix unless more of a comment is needed.  I wasn't sure if people wanted eventually to add a field to the tool database which would indicate tool type.

Reply:
Make sense. :smile: